### PR TITLE
Added script to run different celo nodes in parallel

### PR DESCRIPTION
**Pull request by H34D**
### Description

Run nodes easily with this script. It supports setting of:
- different ports
- sync mode 
- networks

This allows to run different nodes in parallel

### Tested

- Tested by running alfajores and integration network nodes
- Tested by running locally built docker containers as well as used containers from gcp

### Issues

Touches https://github.com/celo-org/celo-monorepo/issues/1807

### CODE REVIEW

File: run-node.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+   --debug \
+   --port=30303 \
+   --nodiscover \
+   --rpcport=8545 \
+   --rpcvhosts=* \
+   --networkid="$NETWORK_ID" \
+   --verbosity=3 \
+   --consoleoutput=stdout \
+   --consoleformat=term \
+   --ethstats=${HOSTNAME}:${WS_SECRET:-1234}@localhost:3000 \
```
Comment (nategraf) [inline run-node.sh]: This script should not assume ethstats is running
```suggestion
```

**Comment by H34D** [comment]
Slightly duplication of https://github.com/celo-org/celo-monorepo/blob/master/packages/cli/start_geth.sh

**Comment by H34D** [comment]
thanks for your feedback @nategraf thats a great base for putting all of this to the cli and or celotool

**Comment by H34D** [comment]
closed in favour of implementing all this in the celotool

**Pull request closed without merging**